docs/development-environment.md

```diff
@@ -129,7 +129,7 @@ You'll need all the tools below to work in the Jetpack monorepo.
 		The latest composer in Homebrew at the time of writing is 2.1.x, but we currently require 2.0.x. If you need to downgrade to a specific version, we suggest these instructions:
 
 		```sh
-		composer self-update 2.0.14
+		composer self-update 2.1.8
 		```
```

Comment:
This whole section is obsolete at the moment. Although we might need to bring it back when Composer releases 2.2, depending on how things go.

Reply:
What do you think is best? Updating the PR to rmeoe the section or just closing the PR ?

Reply:
Either remove the section, or make it more generic somehow.